### ISSUE TITLE
The TFTrainer class has been deprecated in the Hugging Face Transformers library.  Instead you can use.

### ISSUE THREAD

**Issue by Nikhilltiwari** [krishnaik06/Huggingfacetransformer#7]
# Compile the model
optimizer = tf.keras.optimizers.Adam(learning_rate=5e-5)
loss = tf.keras.losses.SparseCategoricalCrossentropy(from_logits=True)
model.compile(optimizer=optimizer, loss=loss, metrics=['accuracy'])

# Train the model
model.fit(train_dataset.batch(8), epochs=2, validation_data=test_dataset.batch(16))

# Evaluate the model
evaluation = model.evaluate(test_dataset.batch(16))
print(f"Loss: {evaluation[0]}, Accuracy: {evaluation[1]}")

# Make predictions
predictions = model.predict(test_dataset.batch(16))
predicted_labels = tf.argmax(predictions.logits, axis=1).numpy()

# Compute confusion matrix
cm = confusion_matrix(y_test, predicted_labels)
print(cm)

# Save the model
model.save_pretrained('senti_model')

**Comment by iamdoublea** [comment]
Hey Nikhil, can we use just Trainer from the same library?
